lib/FusionInventory/Agent/Task/Inventory/Generic/Dmidecode/Battery.pm

```diff
@@ -86,4 +93,65 @@ sub _parseDate {
     return;
 }
 
+sub _mergeBatteries {
```

Comment:
In my point of view, this function should be factorized as a mergeContent() case in Inventory core and then we should use mergeContent() everywhere we addEntry() before.
In the case of merging a list of element, we need to identify each uniq element and merge fields on matching element as needed: we then need specific equality test for battery, maybe we could also merge fields specifying a method to use for each fields. Such merging functionality would be more reusable.